getting-started/vs-integration/scaffolders.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Telerik UI for Blazor Code Scaffolders
+page_title: Telerik UI for Blazor Code Scaffolders - Visual Studio Integration
+description: Learn how to add a new component page in Visual Studio via code scaffolder templates.
+slug: getting-started-vs-integration-scaffolders
+position: 10
+---
+
+# Scaffolders
+
+The Telerik Visual Studio extension provides scaffolders for increased developer productivity. They allow you to quickly add component pages to your application and setup the parameters of the component through UI.
+
+## Get the Extension
+
+To use the **Telerik UI for Blazor Scaffolders**, install the Telerik UI for Blazor Visual Studio Extension. @[template](/_contentTemplates/common/general-info.md#vsx-download)
+
+## Add a new component page
+
+1. Right-click on the name of the solution.
+1. From the popup menu select the `Telerik UI for Blazor Project Item`.
+1. From the UI form, select the desired component and set the available parameters.
+2. The scaffolded page will be added in the `Pages` folder.
+
+>caption Add a new component page
+
+## Available Scaffolded Components
+
+| Component |
```

Comment:
Why table instead of a simple bullet list?

Reply:
I like the idea.